pkg/unvanquished_src.dpkdir/ui/beacons.cfg

```diff
@@ -12,10 +12,10 @@ fadeInScale       1.4
 
 hudSize           1
 hudMinSize        0.025
-hudMaxSize        1
+hudMaxSize        0.7
 hudAlpha          0.35
 hudAlphaImportant 0.7
-hudMargin         0.14
+hudMargin         0.14 // must obey hudMaxSize + 2 * hudMargin < hudSize
```

Comment:
Why

Reply:
Fixed the equation which should have said 1 instead of hudSize. The calculation for positioning the beacon does invalid float operations if the beacon does not fit inside the boundaries for drawing.